{"name": "Verify if calculations are correct in subtotal", "status": "failed", "statusDetails": {"message": "AssertionError: Error. The calculations are incorrect\n", "trace": "  File \"/Users/rshche/PycharmProjects/pythonProject/venv/lib/python3.9/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Users/rshche/PycharmProjects/pythonProject/venv/lib/python3.9/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/esw_checkout.py\", line 178, in verify_subtotal_calculations\n    context.app.evo_checkout.verify_calculations()\n  File \"/Users/rshche/Automation/pages/evo_checkout.py\", line 351, in verify_calculations\n    assert sum(final_list) == grand_total, f'Error. The calculations are incorrect'\n"}, "steps": [{"name": "Given Enter eSW test harness", "status": "passed", "start": 1622838948253, "stop": 1622838952331}, {"name": "When Select end-point - SIT76", "status": "passed", "start": 1622838952332, "stop": 1622838952404}, {"name": "And Select JSON: Number of lines - 1; XA/XP - XP", "status": "passed", "start": 1622838952404, "stop": 1622838952627}, {"name": "And Select country - Canada", "status": "passed", "start": 1622838952628, "stop": 1622838953027}, {"name": "And Enter Checkout", "status": "passed", "start": 1622838953027, "stop": 1622838953066}, {"name": "Then Verify if calculations in Subtotal are correct", "status": "failed", "statusDetails": {"message": "AssertionError: Error. The calculations are incorrect\n", "trace": "  File \"/Users/rshche/PycharmProjects/pythonProject/venv/lib/python3.9/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Users/rshche/PycharmProjects/pythonProject/venv/lib/python3.9/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/esw_checkout.py\", line 178, in verify_subtotal_calculations\n    context.app.evo_checkout.verify_calculations()\n  File \"/Users/rshche/Automation/pages/evo_checkout.py\", line 351, in verify_calculations\n    assert sum(final_list) == grand_total, f'Error. The calculations are incorrect'\n"}, "start": 1622838953066, "stop": 1622838957266}], "start": 1622838945618, "stop": 1622838957504, "uuid": "dff79cdd-7c38-4cb6-9f9a-fcd20d7c7a98", "historyId": "552cfe8849a2648abcb763d1d24c584d", "fullName": "Test eSW Checkout UX for Canada Countries in Test Environment: Verify if calculations are correct in subtotal", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Test eSW Checkout UX for Canada Countries in Test Environment"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}